### PR TITLE
Reduce bundle size of `parser-html.js`

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## Description

**Before:**

```
$ node ./scripts/build/build.mjs --file=parser-html.js --print-size
 Building packages 
parser-html.js..................................................179 kB    DONE  
```

**After:**

```
$ node ./scripts/build/build.mjs --file=parser-html.js --print-size
 Building packages 
parser-html.js..................................................156 kB    DONE  
```

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-html/utils/html-elements-attributes.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+const htmlElementAttributes = require("html-element-attributes");
+const mapObject = require("./map-object.js");
+const arrayToMap = require("./array-to-map.js");
+
+const HTML_ELEMENT_ATTRIBUTES = mapObject(htmlElementAttributes, arrayToMap);
+
```
Comment (fisker) [inline src/language-html/utils/html-elements-attributes.js:8]: Can you try use `html-elements-attributes.evaluate.js` as filename to see if the bundle became smaller?

File: src/language-html/utils/html-tag-names.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+const htmlTagNames = require("html-tag-names");
+const arrayToMap = require("./array-to-map.js");
+
+const HTML_TAGS = arrayToMap(htmlTagNames);
+
+module.exports = HTML_TAGS;
```
Comment (fisker) [inline src/language-html/utils/html-tag-names.js:8]: Same here.

File: src/language-html/utils/html-elements-attributes.js (hunk 1 above)
Comment (sosukesuzuki) [reply src/language-html/utils/html-elements-attributes.js:8]: I tried it in my local but bundle size didn't became smaller.
```
 Building packages 
parser-html.js..................................................156 kB    DONE  
```

Comment (fisker) [reply src/language-html/utils/html-elements-attributes.js:8]: Thanks, it's not safe any way, `mapObject` use `Object.create(null)`, but turns to JSON will be an normal object

`HTML_ELEMENT_ATTRIBUTES.prototype` will be truly.

**Pull request merged**